File: Linguagem C++/Estrutura Condicional/Glicose.cpp
#include <bits/stdc++.h>

using namespace std;

int main()
{
    double glicose;

    cout << "Digite a quantidade de glicose: ";
    cin >> glicose;

    if (glicose >=0 && glicose <=100){
        cout << "Normal" << endl;
    }else if (glicose <=140){
        cout << "Elevado" << endl;
    }else {
        cout << "Diabetes" << endl;
    };

    return 0;
}

File: Linguagem Java/Exercicios/Glicose.java
import java.util.Locale;
import java.util.Scanner;

public class Glicose {

	public static void main(String[] args) {
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		double glicose;
		
		System.out.print("Digite a medida da glicose: ");
		glicose = sc.nextDouble();
		
		if (glicose <= 100) {
			System.out.println("Normal");
			
		}else if (glicose <= 140) {
			System.out.println("Elevado");			
		}else {
			System.out.println("Diabetes");			
		};
		
		sc.close();
	}

}
